stack-fruitkid/recipes/role-internalapi.rb

```diff
@@ -0,0 +1,7 @@
+include_recipe 'stack-easybib::role-phpapp'
```

Comment:
Maybe copy role php app to the fruitkid stack, so that we can separate the stack receipes from each other?